StudentOutreach/Models/MessageFilter.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 enum MessageFilter: CaseIterable, Identifiable {
     case notSubmitted, notGraded, scoredMoreThan, scoredLessThan, markedIncomplete, reassigned
-    case courseScoreLessThan, courseScoreMoreThan, courseScoreBetween, courseScoreEmpty, all
+    case courseScoreLessThan, courseScoreMoreThan, courseScoreBetween, courseScoreEmpty, NoCourseActivitySevenDays, all
```

Comment:
For consistency, let's start this with a lowercase letter: `noCourseActivitySevenDays`

Reply:
Made the change